File: 프로그래머스/unrated/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    answer = 0
    
    t_len = len(t)
    p_len = len(p)
    
    idx = 0
    while idx <= t_len - p_len:
        tmp = t[idx : idx + p_len]
        if int(tmp) <= int(p):
            answer += 1
        
        idx += 1
    
    return answer

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    var answer = 0;
    
    for (let i = 0; i <= t.length - p.length; i++) {
        const num = t.slice(i, i + p.length);
        answer += Number(num) <= Number(p) && 1;
    }
    
    return answer;
}